src/map/entities/charentity.cpp

```diff
@@ -927,7 +927,7 @@ void CCharEntity::OnAbility(CAbilityState& state, action_t& action)
         }
 
         // remove invisible if aggresive
-        if (PAbility->getID() != ABILITY_FIGHT)
+        if (PAbility->getID() != ABILITY_TAME || PAbility->getID() != ABILITY_FIGHT)
```

Comment:
This should be `&&` right? If the ability is not either then remove invisible.

Reply:
This is the exact coding that kj told me to write and PR.  Ask him if && would be better...